lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml

```diff
@@ -1,15 +1,20 @@
-{{- include "lumigator.mistral-default-secret" . -}}
```

Comment:
rename this file

Reply:
Yeah, you're right. Instead, I'm going to create a new version of the code with two different secrets (one per each key) in order to provide more flexibility to the user, as we discuss yesterday in the infra meeting.